app/main/views/frameworks.py

```diff
@@ -103,6 +103,43 @@ def framework_services(framework_slug):
     ), 200
 
 
+@main.route('/frameworks/digital-outcomes-and-specialists/declaration/<string:section_id>',
```

Comment:
The reason is so, in the prototype, you can click through the 5 pages without any validation. That's the only meaningful difference.